generator/golang/option.go

```diff
@@ -24,80 +24,84 @@ import (
+	ReserveComments             bool `reserve_comments:"Reserve comments of definitions in thrift file"`
+	NilSafe                     bool `nil_safe:"Generate nil-safe getters."`
+	FrugalTag                   bool `frugal_tag:"Generate 'frugal' tags."`
+	EscapeDoubleInTag           bool `unescape_double_quote:"Unescape the double quotes in literals when generating go tags."`
+	GenerateTypeMeta            bool `gen_type_meta:"Generate and register type meta for structures."`
+	GenerateJSONTag             bool `gen_json_tag:"Generate struct with 'json' tag"`
+	AlwaysGenerateJSONTag       bool `always_gen_json_tag:"Always generate 'json' tag even if go.tag is provided (Disabled by default)"`
+	SnakeTyleJSONTag            bool `snake_style_json_tag:"Generate snake style json tag"`
+	LowerCamelCaseJSONTag       bool `lower_camel_style_json_tag:"Generate lower camel case style json tag"`
+	GenerateReflectionInfo      bool `generate_reflection_info:"This option is no longer used. Please use with_reflection instead."`
+	WithReflection              bool `with_reflection:"Generate reflection info"`
+	EnumAsINT32                 bool `enum_as_int_32:"Generate enum type as int32"`
+	CodeRefSlim                 bool `code_ref_slim:"Generate code ref by given idl-ref.yaml with less refs to avoid conflict"`
+	CodeRef                     bool `code_ref:"Generate code ref by given idl-ref.yaml"`
+	KeepCodeRefName             bool `keep_code_ref_name:"Generate code ref but still keep file name."`
+	TrimIDL                     bool `trim_idl:"Simplify IDL to the most concise form before generating code."`
+	EnableNestedStruct          bool `enable_nested_struct:"Generate nested field when 'thrift.nested=\"true\"' annotation is set to field, valid only in 'slim template'"`
+	JSONStringer                bool `json_stringer:"Generate the JSON marshal method in String() method."`
+	WithFieldMask               bool `with_field_mask:"Support field-mask for generated code."`
+	FieldMaskHalfway            bool `field_mask_halfway:"Support set field-mask on not-root struct."`
+	FieldMaskZeroRequired       bool `field_mask_zero_required:"Write zero value instead of current value for required fields filtered by fieldmask."`
+	ThriftStreaming             bool `thrift_streaming:"Recognize thrift streaming annotation and generate streaming code."`
+	NoDefaultSerdes             bool `no_default_serdes:"Do not generate default thrift serdes code."`
+	NoAliasTypeReflectionMethod bool `no_alias_type_reflection_method:"Refer alias type without generating type related methods in <idl>_reflection.go when 'thrift.alias=\"true\"' annotation is set to types in referred thrift, valid only in 'slim template'"`
+	EnableRefInterface          bool `enable_ref_interface:"Generate Interface field without pointer type when 'thrift.interface=\"true\"' annotation is set to types in referred thrift, valid only in 'slim template'"`
```

Comment:
`thrift.is_interface`, if we all agree to change the name.

Reply:
Is there any problem with ```no_alias_type_reflection_method``` and ```enable_ref_interface```? I am just afraid that they are kinda tedious.

Reply:
I don't have better idea than that lol